src/platform/packages/shared/kbn-workflows/kibana.jsonc

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "shared-common",
+  "id": "@kbn/workflows",
+  "owner": ["@elastic/workflows-eng"],
+  "group": "platform",
+  "visibility": "shared"
```

Comment:
I didn't see the example plugin importing from this package, perhaps it can be `"visibility": "private"`?

Reply:
we do plan for it to be shared across solutions

Reply:
This package might need to be imported by other plugins that will integrate with workflows-management plugin. (@talboren please correct me if I'm wrong)